rust/bin/acars_router/Cargo.toml

```diff
@@ -14,8 +14,8 @@ license = "MIT"
 log = "0.4.17"
 tokio = { version = "1.20.1", features = ["full", "tracing"] }
 chrono = "0.4.22"
-serde = { version = "1.0.143", features = ["derive"] }
```

Comment:
I can't tag the line I'm interested in because you didn't edit it, but we should probably bump the `version` in this file so the CI can kick out a new release.

Reply:
The version for `acars_router` has been bumped to `1.0.10` in this PR.

Because it was in a previous commit and then I merged main into the branch, it's not shown as an edit for some reason.

Reply:
`.10` is the current version that has been released. If we bump to `.11` then I think this is ready to go. It's running like a champ in my test setup.